pkgs/top-level/python-packages.nix

```diff
@@ -21202,11 +21202,11 @@ in {
 
   pyudev = buildPythonPackage rec {
```

Comment:
If you have the time, please also move this package to an external file in `./pkgs/development/python-modules`.